invenio_drafts_resources/services/records/service.py

```diff
@@ -190,8 +206,15 @@ def publish(self, id_, identity, links_config=None):
 
         # Fully validate draft now
         # TODO: Add error_map for `ValidationError` in the resource config
```

Comment:
minor: wondering if this old TODO makes sense now, as far as I understand errors are handled but not as an exception.

Reply:
Right. I will remove those.